### ISSUE TITLE
[BUG] swarms 5.2.7 Agent example

### ISSUE THREAD

**Issue by evelynmitchell** [kyegomez/swarms#519]
Version: 5.2.7
in Colab

Readme example Agents
```
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-2-b43a687f8bdb>](https://localhost:8080/#) in <cell line: 6>()
      4 
      5 # Import the OpenAIChat model and the Agent struct
----> 6 from swarms import Agent, OpenAIChat
      7 
      8 # Load the environment variables
```
...
```
[/usr/local/lib/python3.10/dist-packages/swarms/artifacts/__init__.py](https://localhost:8080/#) in <module>
      1 from swarms.artifacts.base_artifact import BaseArtifact
      2 from swarms.artifacts.text_artifact import TextArtifact
----> 3 from swarms.artifacts.artifact_main import Artifact
      4 
      5 __all__ = [

ModuleNotFoundError: No module named 'swarms.artifacts.artifact_main'
```

I've verified that swarms/artifacts now has base_artifact and text_artifact, neither of which provides, Artifact.

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/kyegomez) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/kyegomez/swarms/issues/519">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/kyegomez/swarms/issues/519/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/kyegomez/swarms/issues/519/pledge.svg">
</picture>
</a>

**Comment by kyegomez** [comment]
@evelynmitchell need to update!

**Comment by evelynmitchell** [comment]
Fixed in 5.3.1 

Thank you!
